Service/UserDiscountCreate.php

```diff
@@ -1,23 +1,84 @@
 <?php
 namespace Zumba\User\Service;
 
-use \Zumba\User\Contract\Command;
+use \Zumba\User\Contract\Command,
+	\Zumba\User\Factory\EventQueueRepo,
+	\Zumba\User\Factory\Repo,
+	\Zumba\User\Contract\Transactional;
 
-class UserDiscountCreate implements Command {
+class UserDiscountCreate implements Command, Transactional {
```

Comment:
This would extend abstract command. The parent class will implement Transactional.